core/lib/dal/src/blocks_dal.rs

```diff
@@ -595,6 +596,16 @@ impl BlocksDal<'_, '_> {
     ) -> anyhow::Result<()> {
         let base_fee_per_gas = BigDecimal::from_u64(miniblock_header.base_fee_per_gas)
             .context("base_fee_per_gas should fit in u64")?;
+        let gas_per_pubdata_limit = if miniblock_header
+            .protocol_version
+            .unwrap_or(ProtocolVersionId::last_pre_boojum())
+            .is_1_4_1()
```

Comment:
I might be missing something, but won't it be a problem when we integrate the next VM version? You check if the current vm is `1.4.1` and **if not** you apply `MAX_GAS_PER_PUBDATA_BYTE_PRE_1_4_1`. When we create 1.5, won't we by mistake use the old value? Compiler won't help here.

Reply:
oh now I see that the implementation of `is_1_4_1` checks if its `1.4.1` **or later**. So it will work correctly, but it is still confusing. Perhaps `is_post_1-4-1`?